common/app/views/fragments/metaData.scala.html

```diff
@@ -8,22 +8,36 @@
 @import views.support._
 
 @* Critical meta data that have an impact on rendering speed *@
-
 <meta http-equiv="X-UA-Compatible" content="IE=Edge" />
-
-<meta name="viewport" content="width=device-width, initial-scale=1.0" />
 <meta name="format-detection" content="telephone=no" />
 <meta name="HandheldFriendly" content="True" />
 
-@* http://www.chromium.org/developers/design-documents/dns-prefetching *@
-<link rel="dns-prefetch" href="@Configuration.assets.path" />
-<link rel="dns-prefetch" href="@Configuration.images.path" />
-<link rel="dns-prefetch" href="@Configuration.ajax.url" />
-<link rel="dns-prefetch" href="@AnalyticsHost()" />
-<link rel="dns-prefetch" href="//j.ophan.co.uk" />
-<link rel="dns-prefetch" href="//ophan.theguardian.com" />
-<link rel="dns-prefetch" href="//oas.theguardian.com" />
-<link rel="dns-prefetch" href="@Configuration.debug.beaconUrl" />
+@if(amp) {
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width,initial-scale=1,minimum-scale=1,maximum-scale=1,user-scalable=no,minimal-ui">
+    <link rel="amphtml" href="/@item.id/amp">
```

Comment:
rememebr to wrap in an `if((item.isArticle && !item.isLiveBlog) || item.isGallery){` as well as moving it